cf-oscore/src/main/java/org/eclipse/californium/oscore/ErrorDescriptions.java

```diff
@@ -58,6 +58,7 @@
 	public static final String STRING_NULL = "String is null";
 	public static final String CONTEXT_NULL = "Context is null";
 	public static final String ALGORITHM_NOT_DEFINED = "Algorithm not defined";
+	public static final String CONTEXT_REGENERATION_FAILED = ("Security context re-generation failed");
```

Comment:
Any reasons for the braces '(" .... ")'

Reply:
I never considered this but some of the strings in ErrorDescriptions have braces and some do not. Should I simply remove the braces from all the strings in the file?

Reply:
Though the `(...)` are obsolete in java, I would prefer, if they are removed.

Reply:
Now changed in commit 059f82c7bf9ff013b8dcbce7d4b47c727d5b0b1c